HW1/P3/P3.py

```diff
@@ -0,0 +1,43 @@
+
+# coding: utf-8
+
+# In[1]:
+
+import findspark
+import os
+findspark.init('/home/chongmo/spark') # you need that before import pyspark.
+import pyspark
+from pyspark import SparkContext
+sc =SparkContext()
+
+
+# In[2]:
+
+import numpy as np
+import matplotlib.pyplot as plt 
+import matplotlib.cm as cm
```

Comment:
You're not actually using matplotlib in P3, no? Not that it makes a huge difference, but removing imports that you're not using makes your code a bit more readable (and might save some time).